lib/bot.js

```diff
@@ -264,6 +264,20 @@ module.exports = (function() {
 			);
 		},
 
+        getPagesInNamespace: function(namespace, callback) {
+            this.log("Getting pages in namespace " + namespace);
+
+            this.api.call({
```

Comment:
Please use `this.getAll` instead (see `getPagesInCategory` above) to iterate over all pages.